### ISSUE TITLE
Website Categoires are shown not in alphabetical order in the filters section and on the dashboard of OONI MAT

### ISSUE THREAD

**Issue by sloncocs** [ooni/explorer#848]
## Expected Behavior
The list of website categories should be similarly structured in alphabetical order in all interfaces.

## Actual Behavior

In the Filters section and in the dashboard, the 'Social Networking' and 'Terrorism and Militants' categories are in the wrong places.

<img width="1230" alt="Screenshot 2023-03-21 at 14 52 27" src="https://user-images.githubusercontent.com/16296135/226586147-a2a7c6c7-ffe2-4339-8230-aae0114abaaf.png">

<img width="1180" alt="Screenshot 2023-03-21 at 14 52 38" src="https://user-images.githubusercontent.com/16296135/226586155-bc0466b6-4f90-40c5-9dfd-874cf5b6a15b.png">

## Steps to Reproduce the Problem

  1. Set 'Rows' to 'Websites Categories'
  2. Scroll the Filters section to find 'Social Networking' and and 'Terrorism and Militants' 
  3. Scroll the dashboard to find 'Social Networking' and 'Terrorism and Militants' 

## Specifications
 NA

**Comment by chidexebere** [comment]
A little investigation shows that the category code was used to query the actual category, filter and sort as shown below.

![MAT-data](https://user-images.githubusercontent.com/25040059/231365420-68165bfa-2fc2-400a-9dbb-8670c705ab0a.jpg)


The category codes for 'Social Networking' and 'Terrorism and Militants' are `GRP` and `MILX` respectively which do not closely follow like others. For example:

- 'Anonymization and circumvention tools' is `ANON`
- 'Government'  is `GOVT`
- 'Hacking Tools' is `HACK`
- 'LGBTQ+' is `LGBT`
- 'Miscellaneous content' is `MISC`

This issue also affects categories like 'E-commerce' and 'Media sharing' because their category codes are `COMM` and `MMED` respectively.

A better solution will be to make the changes to the category codes in the **OONI API** but this will require the core team  because there might be reasons why these conventions were used.

If for some reason these codes can not be easily changed, the sorting algorithm might have to be changed.

**TL;DR**: input from the core team is needed.
